pkgs/checks/README.md

```diff
@@ -44,6 +45,6 @@ checkThat(actual).deepEquals(expected);
 
 ## Differences in behavior from matcher
 
-- The `equals` Matcher performed a deep equality check on collections.
-  `.equals()` check will only correspond to [operator ==] so some tests may need
-  to replace `.equals()` with `.deepEquals()`. **TODO: implement `deepEquals`**
+-   The `equals` Matcher performed a deep equality check on collections.
```

Comment:
Indent diff intentional here?

Reply:
I think I probably autoformatted and this is what the internal formatter does.